src/bundle/Resources/public/js/scripts/sidebar/side.panel.js

```diff
@@ -20,6 +20,9 @@
                 sidePanel.classList.add(CLASS_HIDDEN);
                 doc.body.removeEventListener('click', handleClickOutside, false);
                 removeBackdrop();
+                if (global.location.href.includes('notifications/render/all')) {
```

Comment:
new line before

And I'm not sure if we can use hardcoded string url for this check, needs someone else expertise